pages/Documents/Security&Authentication/ConsumerAuthentication/configuration.md

```diff
@@ -21,33 +21,40 @@ indicator: both
 
 ![Search for Consumer Identity Provider in the Management Console](img/mgmt_search_consumer_idp.png)
 
+![Consumer Identity Provider in the Management Console](img/mgmt_consumer_idp.png)
+
+
 {:start="3"}
 3.  Click on 'Click to configure'
 
 ![Click to configure the Consumer Identity Provider](img/mgmt_consumer_idp_configure.png)
 
 {:start="4"}
-4. In section #2 — "Define Identity Provider", select the preferred OAuth 2.0 Authentication Type, and then complete the required fields
+4. In section #2 — "Configure identity providers auth settings", select the preferred OAuth 2.0 Authentication Type, and then complete the required fields
 
 ![Select consumer authentication type](img/consumer_idp_select_type.png)
 
 The following parameters should be defined:
 
 * **OAuth 2.0 Authentication Type** - Implicit / Code Flow
 
-* **OAuth 2.0 Authentication Endpoint** - An HTTPS URL that used in the case that the consumer conversation window is configured as a [Separate Browser Window](https://knowledge.liveperson.com/messaging-channels-web-messaging-authenticated-web-messaging.html#separate-browser-window-flow).
+* **Issuer Display Name** - The presentation name for the identity provider issuer
 
-* **OAuth 2.0 Token Endpoint** - An HTTPS URL that used in Code Flow
+* **JWT issuer (iss)** - The identifier of the principal that issued the JWT. The 'iss' value is a case-sensitive string containing a URL value.
 
-* **Choose IDP Certificate Type** - JWT Public Key / JWKS Endpoint
+* **Authentication Endpoint** - An HTTPS URL OAuth 2.0 Authorization Endpoint that is used by LivePerson to initiate the authentication flow with your idenitity principal. (this endpoint is invoked in case the conversation window opens as an external window)
```

Comment:
Either move the parentheses to within the sentence, or make it a separate sentence by starting with “This” and ending the sentence with a period.